concordium-consensus/src/Concordium/Queries.hs

```diff
@@ -459,10 +491,10 @@ getBlockBirkParameters = liftSkovQueryBlock $ \bp -> do
 
 -- |Get the cryptographic parameters of the chain at a given block.
 -- The result is versioned (which will currently always be version 0).
```

Comment:
Outdated comment. 
What was the reasoning behind not returning `Versioned` anymore?

Reply:
There is only one version